collector/topological_inventory.py

```diff
@@ -235,10 +318,20 @@ def worker(_: str, source_id: str, dest: str, b64_identity: str) -> None:
                 return
 
         LOGGER.info(
-            '%s: %s: %s\t%s',
-            thread.name, source_id, entity, len(all_data)
+            '%s: %s: Account# %s: %s\t%s',
+            thread.name, source_id, tenant, entity, len(all_data)
         )
-        data['data'][entity] = all_data
+
+        if all_data:
+            data['data'][entity] = all_data
+        else:
+            LOGGER.debug(
+                '%s: !!Inadequate Topological Inventory data for '
```

Comment:
Why is this debug message full of exclamation marks? It's either important - then it should be a higher log level, or it's really a debug message - then no exclamation mar should be needed.

Reply:
It's an important message that deserves attention.
But at the same time, it would be hard to miss.
I will remove the exclamations